script/foreman-puma-status

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+scl enable tfm 'puma-status /var/run/foreman/pids/puma.state'
```

Comment:
No `scl enable` needed here. We treat this like a standard bash script so it works everywhere. Then our packaging environments can adjust this as needed. For example, in EL7 environments, we update this to change the entrypoint:

```
#!/bin/bash
```

becomes

```
#!/usr/bin/tfm-ruby
```

Reply:
ok I removed `scl` and made it like a simple bash script.